db/sql/rounds/restore_votes_snapshot.sql

```diff
@@ -20,6 +20,6 @@
 */
 
 UPDATE mem_accounts m
-SET vote = b.vote
+SET vote = b.vote, "missedBlocks" = b."missedBlocks", "producedBlocks" = b."producedBlocks"
 FROM mem_votes_snapshot b
```

Comment:
As you are refactoring, I suggest to use `FROM mem_votes_snapshot snapshot` and `UPDATE mem_accounts accounts`. That will make the query more readable like `SET accounts.vote = snapshot.vote`.